py/defs.bzl

```diff
@@ -90,6 +90,7 @@ def _py_binary_or_test(name, rule, srcs, main, deps = [], resolutions = {}, **kw
         package_collisions = package_collisions,
         tags = ["manual"],
         testonly = kwargs.get("testonly", False),
+        target_compatible_with = kwargs.get("target_compatible_with", False),
```

Comment:
I think we want the default here to either be `[]` or `None`.

Reply:
yes! I copied from the line above and missed that thanks